diagnostic.md

```diff
@@ -73,14 +91,31 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl  http://www.example.com/donuts/ \
```

Comment:
Just An FYI, these two examples are verbatim from the repos we have from the API talks. The only thing I modified is the base URL string. In the examples, one line curl scripts put the URL at the end, but multi line scripts put the URL on the first line after curl but before the first backslash.